definition-map.js

```diff
@@ -4,6 +4,19 @@ const TYPES = {
         fields: {
             contentTypes: "Content Types requested in the Catalog."
         }
+    },
+    SuperQuiz: {
+        metadescription: "Contains quiz questions and other data about the entered quiz(s).",
+        fields: {
+            quizzes: "The IDs of the Quizzes user selects and has access to.",
+            questions: "The collection of Quiz questions user selects.",
+            totalTimeInSeconds: "The total estimated time for all of the Quiz questions. This is only applicable when all the selected Quizzes enable the `Estimated Time Per Question` option from the user's Thought Industries instance.",
```

Comment:
Would note that with topic creation GraphQL Resolvers the user wouldn't necessarily need to set in the Thought Industries instance. Would maybe update the wording to something like: "...only applicable when all the Quizzes have the `Estimated Time Per Question` option enabled."